emails/views.py

```diff
@@ -40,13 +39,15 @@ def _get_data_from_request(request):
 
 @csrf_exempt
 def index(request):
+    incr_if_enabled('emails_index', 1)
     request_data = _get_data_from_request(request)
     if (not request.user.is_authenticated and
         not request_data.get("api_token", False)
        ):
         raise PermissionDenied
     if request.method == 'POST':
         return _index_POST(request)
+    incr_if_enabled('emails_index_GET', 1)
```

Comment:
```suggestion
    incr_if_enabled('emails_index_get', 1)
```
It seems that metric name uses lower cases. Like this [post](https://github.com/mozilla/fx-private-relay/pull/451/files#diff-2c3a9c773eb79ee1e4b1858c23620e74R61), [put](https://github.com/mozilla/fx-private-relay/pull/451/files#diff-2c3a9c773eb79ee1e4b1858c23620e74R106), and [delete](https://github.com/mozilla/fx-private-relay/pull/451/files#diff-2c3a9c773eb79ee1e4b1858c23620e74R123).